tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -1447,6 +1447,19 @@ def test_generate_messages_with_defaults():
             "ge_version": "0.11.5.manual_testing",
         },
     ],
+    "expectation_suite.add_expectation": [
+        {
+            "event_payload": {},
+            "event": "expectation_suite.add_expectation",
+            "success": True,
+            "version": "1.0.0",
+            "event_time": "2020-06-25T16:08:28.070Z",
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.13.18.manual_testing",
```

Comment:
We've been using the convention that added messages use the `ge_version` corresponding to when they were released - so this would be 0.13.47, the next version
```suggestion
            "ge_version": "0.13.47.manual_testing",
```